test/test_flake8.py

```diff
@@ -12,12 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from ament_flake8.main import main
+from flake8.api.legacy import get_style_guide
 import pytest
 
 
 @pytest.mark.flake8
 @pytest.mark.linter
 def test_flake8():
-    rc = main(argv=[])
-    assert rc == 0, 'Found code style errors / warnings'
+    style = get_style_guide(
+        ignore='D100,D103,D104',
+        # flake8 doesn't appear to honor the import_order_style directive via this api.
+        # https://github.com/ros-infrastructure/catkin_pkg/pull/219#issuecomment-396092843
+        import_order_style='google',
```

Comment:
Can the option as well as the comment be removed since it doesn't have any effect anyway?